is_all_unique_elements.js

```diff
@@ -0,0 +1,16 @@
+
+const isAllUniqueElements = (array) => {
+  let isAllUnique = true;
+
+  array.forEach((currentElement, currentIndex) => {
+    array.slice(currentIndex + 1).forEach((elementToCompare) => {
+      if (elementToCompare === currentElement) {
+        isAllUnique = false;
```

Comment:
We can add a return if isAllUnique = false instead to make the program run more efficiently